docs/docs/services/qix-engine/access-control.md

```diff
@@ -148,7 +260,7 @@ resource.resourcetype = "*" and user.employeeType = "developer" and resource.act
 This operator compares two expressions and returns `true` only if both evaluate
 to `true`.
 
-**Syntax**
+##### Syntax
 
 ```c
 (EXPRESSION) && (EXPRESSION)
```

Comment:
Are these operators subject to lazy evaluation?

Reply:
Yes, expresisons are evaluated lazily last time I checked. As mentioned elsewhere, we need the engine team to verify these questions.